ui/app/src/Views/Settings/Types.elm

```diff
@@ -0,0 +1,10 @@
+module Views.Settings.Types exposing (..)
+
+
+type alias Model =
+    { startOfWeek : Int
```

Comment:
Can we call this the same name as elsewhere, `firstDayOfWeek`, and use the custom type instead of the int?